addons/velero/1.5.1/tmpl-restic-daemonset-proxy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: restic
+spec:
+  template:
+    spec:
+      containers:
+        - name: restic
+          env:
+          - name: HTTP_PROXY
+            value: ${PROXY_ADDRESS}
+          - name: NO_PROXY
+            value: ${NO_PROXY_ADDRESSES},${OBJECT_STORE_CLUSTER_IP},${KUBERNETES_CLUSTER_IP}
```

Comment:
`NO_PROXY_ADDRESSES` includes the service cidr and both restic and velero are written in Go so it should be redundant to add the two cluster IPs.